src/main/kotlin/io/meshcloud/dockerosb/metrics/gauge/PaginatedGaugeMetricsProvider.kt

```diff
@@ -0,0 +1,20 @@
+package io.meshcloud.dockerosb.metrics.gauge
+
+import io.meshcloud.dockerosb.metrics.MetricsProvider
+import io.meshcloud.dockerosb.metrics.ServiceInstanceDatapoints
+import java.time.Instant
+
+/**
+ * Provides gauge metrics with one page containing the metrics data for a single instance
+ */
+interface PaginatedGaugeMetricsProvider : MetricsProvider<PaginatedGaugeMetricsProvider> {
+  /**
+   * index: the index of the service instance that the metrics is request is for, after sorting by last modified time
```

Comment:
```suggestion
   * index: the page index of the metrics result set for this service definition. Results are sorted by last modified time.
```

Reply:
I guess the interface wasn't clear about how these parameters should work. I added explanations for all of them.